performance/src/main/java/uk/gov/gchq/palisade/example/perf/PerfApplication.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.palisade.example.perf;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
+
+/**
+ * Main class for the performance testing tool.
+ */
+
+@SpringBootApplication
+@EnableDiscoveryClient
+public class PerfApplication {
+    @Autowired
+    CommandLineRunner runner;
```

Comment:
Shouldn't really Autowire, can a bean be configured instead?

Reply:
Not needed, left in from debugging stuff